### PR TITLE
Update/params

### PR DESCRIPTION
**Pull request by mattheww95**
Updated ECTyper parameters to have defaults set in the configuration file, and for SISTR as well. These options now match with the defaults used in IRIDA.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -40,6 +40,8 @@ Initial release of phac-nml/mikrokondo. Mikrokondo currently supports: read trim
 
 ### `Fixed`
 
+- Matched ECTyper and SISTR parameters to what is set in the current IRIDA addressing issue [#58](https://github.com/phac-nml/mikrokondo/issues/58)
```
Comment (apetkau) [inline CHANGELOG.md:43]: Once https://github.com/phac-nml/mikrokondo/pull/66 gets merged in, can you merge `dev` into this branch and move this `Fixed` bullet point to the **Unreleased** section.

**Review by apetkau — CHANGES_REQUESTED** [review]
This looks good. Just one comment, which requires the other PR to be merged.

File: CHANGELOG.md (hunk 1 above)
Comment (mattheww95) [reply CHANGELOG.md]: fixed in: https://github.com/phac-nml/mikrokondo/pull/68/commits/9beb0f111c0d4495ac98e3e0336116b954e27257

**Review by apetkau — APPROVED** [review]
Looks great. Thanks so much @mattheww95

**Pull request merged**